### ISSUE TITLE
React native build failing with new arch (fabric) enabled on android

### ISSUE THREAD

**Issue by ashtar23** [f-23/react-native-passkey#43]
Hello, I've been recently trying to enable the new architecture but came accross multiple packages that seem to have issues with it and are failing to build on android. One of which seems to be this one.

Even with the latest beta version `"react-native-passkey": "3.0.0-beta2"` the build is failing

<img width="763" alt="Screenshot 2024-08-06 at 6 22 32 PM" src="https://github.com/user-attachments/assets/589534dc-aa1e-4f49-9240-38fc40f06529">

Here's a minimal reproducible [example](https://github.com/ashtar23/AwesomeProject)

Here's the log too
[rn-log.log](https://github.com/user-attachments/files/16513148/rn-log.log)

**Comment by ashtar23** [comment]
Just to follow up on this. in `settings.gradle` this seems to be the issue

```
react {
    jsRootDir = file("../src/") // there is no src directory since it's not being included
    libraryName = "Passkey"
    codegenJavaPackageName = "com.reactnativepasskey"
  }
```
So in `package.json` maybe adding "src" should do be enough
```
 "files": [
    "src",
    "lib",
...
```

**Comment by f-23** [comment]
Thank you for the suggestion!

I'll take a look and fix it for beta 3.

**Comment by f-23** [comment]
@ashtar23 This should be fixed in 3.0.0-rc, I would appreciate feedback if this has solved ur issue. Thank you for bringing this up and the proposed solution!

**Comment by ashtar23** [comment]
@f-23 hey, yeah sure thing. I'll give the rc a go, thanks!

**Comment by f-23** [comment]
I'll close this issue as the build errors should be resolved with 3.0. Feel free to open the issue again if you encounter any more problems.
